Update Social Links and APIs

nzec opened the pull request:
* Added Flickr API and Twitter API with FOSSASIA account.
* Added FOSSASIA Social and Web Links
* Updated FontAwesome (Fonts as well as CSS)
* Added fossasia-light.png

abishekvashok commented on index.html:
```diff
@@ -1,5 +1,3 @@
----
```
These are required for jekyll.

nzec replied:
@abishekvashok Oh okay. And as for the icons not rendering, please let it load completely.

nzec replied:
@abishekvashok Fixed.

abishekvashok commented:
Typo in the PR title

abishekvashok commented:
Can you double check, the icons are not rendering.
![screenshot from 2017-12-01 13-29-21](https://user-images.githubusercontent.com/8947010/33473264-b36039b6-d69b-11e7-839e-3c334d2664a6.png)

abishekvashok commented:
Still the same @duttaditya18 

![screenshot from 2017-12-01 13-36-35](https://user-images.githubusercontent.com/8947010/33473504-af190f4e-d69c-11e7-956b-362cfed5b1ea.png)

duttaditya18 commented:
@abishekvashok I fixed it. Check it out now.

harshit98 commented:
@duttaditya18 Kindly provide screenshot of the fix. Don't hurry. Take your time. :)
Nice work! 🎉

abishekvashok reviewed (approved):
LGTM at: https://duttaditya18.github.io/gci17.fossasia.org/

CosmicCoder96 commented:
@duttaditya18 Again, please squash the commits :)

abhinavk96 reviewed (changes requested):
@duttaditya18 Again, please squash the commits :)

abhinavk96 commented on index.html:
```diff
@@ -6,7 +6,7 @@
         <title>FOSSASIA | Google Code-In 2017</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
+        <link href="css/https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all">
```
@duttaditya18 that's not how it's supposed to be, please check.

nzec replied:
Fixed.

duttaditya18 commented:
@harshit98 The Flickr fix does not work on Github Pages because the API is synced with the gci17.fossasia.org website. You need to download it to see. It will work when you put it up. 

@CosmicCoder96 I do not know how to and I messed up the previous repo by trying to do so from GitKRaken.

And here are the screenshots : https://imgur.com/a/xdrJn

CosmicCoder96 commented:
@duttaditya18 Please use git command line instead of git GUI gitKraken etc. There are plenty of resources available to learn how to squash commits on git. try.github.io is where you should start :)

duttaditya18 commented:
@CosmicCoder96 Okay. So I have merged my commits. So can now this be merged?

harshit98 commented:
@duttaditya18 The pull request is still showing 3 commits. Is there any issue you're facing ? Let me know.

CosmicCoder96 commented:
@harshit98 no, the commits are squashed now. Refresh.

abhinavk96 reviewed (approved):
LGTM!

duttaditya18 commented:
@CosmicCoder96 I had to resolve the conflicts, so there had to be 2 more commits.

duttaditya18 commented:
Just merge this already!

jigyasa-grover commented:
@duttaditya18  Kindly be patient and perhaps squash your 5 commits into 1 ? :)

duttaditya18 commented:
Okay @jigyasa-grover If I resolve this conflict, there will be another commit. Can't this be just merged already?

abishekvashok commented:
@jigyasa-grover since, @duttaditya18 is new, can we use the squash and merge button?

duttaditya18 commented:
@abishekvashok @jigyasa-grover I have resolved all the conflicts. It would be awesome if this is merged.

jigyasa-grover commented:
@duttaditya18 Resolve the conflicts pls, we'd "squash & merge", thanks !

duttaditya18 commented:
@jigyasa-grover I have resolved all the conflicts. Merge this before more conflicts arise.

duttaditya18 commented:
@jigyasa-grover @abishekvashok So will this be merged now? Like please. Just do it already.

nzec committed:
Update Social Lins and APIs

* Added Flickr API and Twitter API with FOSSASIA account.
* Added FOSSASIA Social and Web Links
* Updated FontAwesome (Fonts as well as CSS)
* Added fossasia-light.png
<Your commit message> (#44)


Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Merge remote-tracking branch 'origin/gh-pages' into gh-pages


Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Merge remote-tracking branch 'origin/gh-pages' into gh-pages


Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Merge remote-tracking branch 'origin/gh-pages' into gh-pages


Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages
Blog Section and Thank You For Contributing (#64)


added photo of yashkumarverma (#57)


Merge branch 'gh-pages' into gh-pages
Merge branch 'gh-pages' into gh-pages

nzec committed:
Merge branch 'gh-pages' into gh-pages

nzec committed:
Merge branch 'gh-pages' into gh-pages

abishekvashok commented:
@duttaditya18 please wait, I don't have write access, let's wait for a someone with write access to squash and merge this.

Ping @fossasia/fossasia-gci

duttaditya18 commented:
Link to my task: https://codein.withgoogle.com/dashboard/task-instances/6633250158018560/
